tests/xml/M-CR603-formMainZone_MainZoneXml.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<item>
+<Power><value>ON</value></Power>
+<ZonePower><value></value></ZonePower>
+<InputFuncList>
+<value>SOURCE</value>
+<value>TUNER</value>
+<value>PHONO</value>
+<value>CD</value>
+<value>DVD</value>
+<value>BD</value>
+<value>TV</value>
+<value>SAT/CBL</value>
+<value>GAME</value>
+<value>DOCK</value>
+<value>DVR</value>
+<value>V.AUX</value>
+<value>NET/USB</value>
+<value>SIRIUS</value>
+<value>HDRADIO</value>
+<value>NET/USB</value>
+<value>RHAPSODY</value>
+<value>NAPSTER</value>
+<value>PANDORA</value>
+<value>LASTFM</value>
+<value>IRADIO</value>
+<value>SERVER</value>
+<value>USB/IPOD</value>
+<value>USB</value>
+<value>AUXA</value>
+<value>AUXB</value>
+<value>AUXC</value>
+<value>AUXD</value>
+<value>M-XPORT</value>
+<value>IPOD</value>
+</InputFuncList>
+<RenameSource>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+<value></value>
+</RenameSource>
+<RenameZone><value></value></RenameZone>
+<SourceDelete>
```

Comment:
@bmesuere: according to this SourceDelete section you deleted each input source on your receiver. The error message with the missing mapping for the TUNER is raised, because it was deleted.
Please check if you deleted (deactivated) all sources in your web interface or if this is a bug